content/index.es.md

```diff
@@ -62,67 +66,70 @@ Enlaces rápidos a recursos:
+Las WCAG se dirigen principalmente a:
 
-Hay criterios de conformidad adicionales en 2.1 que no existen en 2.0. Se hace una presentación de ellos en las [[novedades de WCAG 2.1]](/standards-guidelines/wcag/new-in-21/).
+-   Desarrolladores de contenido web (autores de páginas, diseñadores de sitios, etc.)
+-   Desarrolladores de programas para crear contenido web
+-   Desarrolladores de herramientas para la evaluación de la accesibilidad web
+-   Otros que quieran o necesiten un estándar para la accesibilidad web, incluyendo la accesibilidad móvil
 
-**El contenido que es conforme con WCAG 2.1 también lo es con WCAG 2.0** (se suele decir que se da “retrocompatibilidad”). Un sitio web que cumpla con WCAG 2.1 cumpliría con los requisitos de las políticas que hacen referencia a WCAG 2.0.
+Para cubrir las necesidades de otros &mdash; incluyendo legisladores, directores e investigadores &mdash; existen diferentes [recursos de la WAI](/resources/).
 
-Dicho de otra forma: si se desea cumplir tanto con WCAG 2.0 y 2.1, se pueden usar los recursos de 2.1 y no sería necesario consultar los relativos a 2.0.
+## Qué hay en las WCAG 2 {#whatis2}
 
-Tanto WCAG 2.0 como 2.1 son estándares actuales. WCAG 2.1 no deja WCAG 2.0 obsoleto ni lo sustituye. Desde W3C se anima a usar la versión más reciente de WCAG al desarrollar o actualizar contenido o políticas de accesibilidad.
+WCAG 2.2 tiene 13 pautas. Las pautas están organizadas bajo [cuatro principios: perceptible, operable, comprensible y robusto](https://www.w3.org/WAI/WCAG22/Understanding/intro#understanding-the-four-principles-of-accessibility).
 
-## Destinatarios de WCAG {#for}
+Para cada pauta existen *criterios de conformidad* que se pueden comprobar. Estos criterios se clasifican en [tres niveles: A, AA y AAA](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels).
 
-WCAG se dirige principalmente a:
+Los criterios determinan la "conformidad" con las WCAG. Es decir, que para cumplir con las WCAG, el contenido tiene que cumplir con los criterios. Puede encontrar más detalles en la [sección sobre la conformidad de las WCAG](https://www.w3.org/TR/WCAG22/#conformance).
 
--   Desarrolladores de contenido web (autores de páginas, diseñadores de sitios, etc.)
--   Desarrolladores de programas de autor
--   Desarrolladores de herramientas para la evaluación de la accesibilidad web
--   Otros que quieran o necesiten un estándar para la accesibilidad web, incluyendo la accesibilidad móvil
-
-Los recursos relacionados se dirigen a cubrir las necesidades de diferentes personas, entre ellas legisladores, directores, investigadores y otros.
-
-WCAG es un estándar técnico, no una introducción a la accesibilidad. Si busca material introductorio, consulte  [“¿Dónde debería empezar?” en las FAQ](/standards-guidelines/wcag/faq/#start).
+Para un resumen breve sobre las pautas WCAG 2, consulte **[["WCAG 2 de un vistazo"]](/standards-guidelines/wcag/glance/es)**.
```

Comment:
```suggestion
Para un resumen breve sobre las pautas WCAG 2, consulte **[["WCAG 2 de un vistazo"]](/standards-guidelines/wcag/glance/)**.
```

@cmuncharaz Manually pointing to a WAI resource translation is not recommended: we have a plugin that automatically point to the translated version when existing.